saul-core/doc/MODELS.md

```diff
@@ -1,6 +1,101 @@
-* Designing flexible learning models including various configurations such as:
 
-   * Local models i.e. single classifiers. (Learning only models (LO)).
-   * Constrained conditional models (CCM)[1] for training independent classifiers and using them jointly for global decision making in prediction time. (Learning+Inference (L+I)).
-   * Global models for joint training and joint inference (Inference-Based-Training (IBT)).
-   * Pipeline models for complex problems where the output of each layer is used as the input of the next layer.
+//Documented by Parisa Kordjamshidi
+
+Designing flexible learning models including various configurations such as:
+
+   * [Local models](#local) i.e. single classifiers. (Learning only models (LO)).
+   * [Constrained conditional models (CCM)](#L+I)[1] for training independent classifiers and using them jointly for global decision making in prediction time. (Learning+Inference (L+I)).
+   * [Global models](#IBT) for joint training and joint inference (Inference-Based-Training (IBT)).
+   * [Pipeline models](#pipeline) for complex problems where the output of each layer is used as the input of the next layer.
```

Comment:
How about replacing "layer" with "model"? (we never talked about "layer" before.)